src/components/grid-aware/StatsBlock/StatsBlock.stories.jsx

```diff
@@ -3,33 +3,39 @@ import StatsBlock from "./StatsBlock";
 
 export default {
   title: "Grid-Aware/StatsBlock",
+  subtitle: "Optional Subtitle",
   component: StatsBlock,
 };
 
-const Template = ({ title, statCards }) => (
-  <StatsBlock title={title} statCards={statCards} />
+const Template = ({ title, subtitle, statCards }) => (
+  <StatsBlock title={title} subtitle={subtitle} statCards={statCards} />
```

Comment:
Since you added `titleColor` as a prop to the component and as an `arg` to the Storybook story, you'll also need to add it here. Otherwise, when you view the Storybook story, the title is still black despite the fact that the controls at the bottom say "blue".